src/envoy/mixer/config.cc

```diff
@@ -69,9 +78,11 @@ void MixerConfig::Load(const Json::Object& json) {
 
   ReadString(json, kNetworkFailPolicy, &network_fail_policy);
 
-  ReadString(json, kDisableCheckCache, &disable_check_cache);
-  ReadString(json, kDisableQuotaCache, &disable_quota_cache);
-  disable_tcp_check_calls = json.getBoolean(kDisableTcpCheckCalls, false);
+  disable_check_cache = ReadBoolean(json, kDisableCheckCache, false);
+  disable_quota_cache = ReadBoolean(json, kDisableQuotaCache, false);
+  disable_report_batch = ReadBoolean(json, kDisableReportBatch, false);
+
+  disable_tcp_check_calls = ReadBoolean(json, kDisableTcpCheckCalls, false);
```

Comment:
nit: we don't need the ReadBoolean function because the json.getBoolean(name, default) automatically returns default if name is not present.

Reply:
Done